man/dnstable_lookup.1

```diff
@@ -113,6 +113,11 @@ presentation format\&. One JSON\-formatted entry per line will be printed\&.
 When using a JSON format for output, add a hex representation of the raw rdata and the rrtype\&.
 .RE
 .PP
+\fB\-s\fR
```

Comment:
Option needs to be updated in the manual page, also.

Reply:
Done. I note the newly added -C case insensitive is missing in the man-page and corresponding .txt file.